.buildkite/dockerfiles-pipeline.yml

```diff
@@ -0,0 +1,124 @@
+definitions:
+  steps:
+    - step: &test-agents
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          useVault: true
+          image: family/enterprise-search-ubuntu-2204-connectors-py
```

Comment:
we don't need this here, right?

Reply:
Correct, this was copied over from the main pipeline and is now removed - thanks for pointing it out.